moped-toolbox/amd_milestones_backfill/create_milestones.py

```diff
@@ -0,0 +1,104 @@
+"""The script inserts moped_proj_milestones into signal and PHB projects. It was created
+to backfill project milestones after implementing the milestone template feature for
+AMD: https://github.com/cityofaustin/atd-data-tech/issues/9102.
+
+Usage:
+    python create_milestones.py -e local -d 2023-01-01
+"""
+import argparse
+
+from utils import get_logger, make_hasura_request, TEMPLATE_MILESTONES
+from queries import PROJECTS_QUERY, PROJ_MILESTONES_MUTATION
+from secrets import HASURA
+
+
+def main(env, max_date_added):
+
+    logger.info(f"Gettings projects added on or before {max_date_added} in {env} env")
+
+    data = make_hasura_request(
+        query=PROJECTS_QUERY,
+        variables={"max_date_added": max_date_added},
```

Comment:
I noticed that the `-d` arg didn't work as expected for projects that I started creating at `2022-06-15T19:06:18.128427+00:00`. I had to pass `2022-06-16` for them to be picked up. Just holding off on approval because of that since it seems having the target date range query as expected is important here. I could be missing something so please let me know if so!

<img width="1256" alt="Screen Shot 2022-06-15 at 2 08 54 PM" src="https://user-images.githubusercontent.com/37249039/173912552-e807ed5b-2cc6-47a9-8299-45409d0c446a.png">

Reply:
thanks for banging on that @mddilley! i thought i understood the expected behavior with taking the shortcut of a partial timestamp, and i clearly didn't.

i'm going to drop this check altogether and update the docs to reflect that the input date string is going to be handled according to postgres's [parsing rules](https://www.postgresql.org/docs/current/datetime-input-rules.html) and suggest that you manually test the graphql query before running this in prod.